src/ui/directives/codemirror/codemirror.js

```diff
@@ -82,8 +82,7 @@ angular.module('app').directive('codemirror', [
         }
 
         function _showAutoComplete(cm, event) {
-          if (cm.state.completionActive) return;
-          if (event && event.keyCode === 13) return;
+          if (event.keyIdentifier.indexOf('Up') !== -1 || event.keyIdentifier.indexOf('Down') !== -1) return;
```

Comment:
:+1: much better, before the autocomplete menu would hide every other character.